src/main/scala/io/iohk/ethereum/mining/Miner.scala

```diff
@@ -203,4 +216,11 @@ object Miner {
   val MaxNonce: BigInt = BigInt(2).pow(64)
 
   val DagFilePrefix: ByteString = ByteString(Array(0xfe, 0xca, 0xdd, 0xba, 0xad, 0xde, 0xe1, 0xfe).map(_.toByte))
+
+  sealed trait MineResult {
```

Comment:
Why is it `MineResult`, not `MiningResult`?  "Mine" is not an action, so this sounds a bit weird to me. Same for the subclasses.

Reply:
('Mine' is an action. They mine ETC. Sounds ok to me.)

Reply:
A verb is not an action, it describes one. We need a noun. When you compute something, do you get a _compute result_ or a _computation result_?

Reply:
`MineResult` would be fine as a command. Maybe it sounds ok to you because you misconstrued it as such? This is why I think it's important to address this.

Reply:
In case you haven't noticed, I changed it :)

Reply:
Yeah, I only noticed after a while. GH pages are refreshing somewhat strangely when you keep them open.

Reply:
Oh boy. Glad I missed this.